R/exercise_1.Rmd

```diff
@@ -227,10 +161,13 @@ xml2::xml_structure(
 )
 ```
 
+Each instance of a given resource will be _roughly_ the same (assuming the FHIR server is working properly), so looking at the structure of just one resource typically provides _most_ of the information you'll need to extract the data from _all_ the resources. By incrementing the `[[1]]` to `[[2]]`, etc. in the commands above you can look at additional resource instances.
```

Comment:
Fix mistake with how to pull specific resources -- need both `[[1]]` and `entry[n]`